extensions/wikia/Chat2/js/views/views.js

```diff
@@ -580,10 +580,11 @@ var NodeChatUsers = Backbone.View.extend({
 		// admin actions
 		if (actions.admin && actions.admin.length) {
 			var adminActions = ul.clone().addClass('admin-actions');
-			for (var i in actions.admin) {
-				var action = actions.admin[i];
+			for (var i in actions.admin) { 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "i" as this name is already used in declaration at line 553. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2814)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Restrict what this loop acts on by testing each property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AForIn)